webui/src/pages/setup/index.jsx

```diff
@@ -54,48 +54,57 @@ server:
+                                Here are the credentials for your first user:<br/>
                             </Card.Text>
-                            <hr/>
-                            <Table borderless hover>
-                                <tbody>
-                                <tr>
-                                    <td>Key ID</td>
-                                    <td><code>{setupData.access_key_id}</code> <ClipboardButton variant="link" text={setupData.access_key_id} tooltip="Copy key ID"/></td>
-                                </tr>
-                                <tr>
-                                    <td>Secret Key</td>
-                                    <td><code>{setupData.secret_access_key}</code> <ClipboardButton variant="link" text={setupData.secret_access_key} tooltip="Copy secret key"/></td>
-                                </tr>
-                                </tbody>
-                            </Table>
-                            <Card.Text>
-                                Download the initial client configuration under your <code>$HOME/.lakectl.yaml</code> and keep a copy of the data for your login into the system<br/>
-                            </Card.Text>
-                            <Alert variant="warning">
+                            <div className={"ml-2 row mt-4"}>
+                                <div className={"col-3"}>Access Key ID:</div>
+                                <div className={"col-7"}><code>{setupData.access_key_id}</code> &#160;&#160;<ClipboardButton className={"copy-button"} variant="outline-dark" text={setupData.access_key_id} tooltip="Copy"/></div>
+                            </div>
+                            <div className={"ml-2 row mt-2"}>
+                                <div className={"col-3"}>Secret Access Key:</div>
+                                <div className={"col-7"}><code>{setupData.secret_access_key}</code> &#160;&#160;<ClipboardButton className={"copy-button"} variant="outline-dark" text={setupData.secret_access_key} tooltip="Copy"/></div>
+                            </div>
+                            <Alert className={"mt-4"} variant="warning">
                                 This is the <strong>only</strong> time that the secret access keys can be viewed or downloaded. You cannot recover them later.
                             </Alert>
-                            <hr/>
-                            <Button variant="success" href={downloadContent} taget="_blank" download="lakectl.yaml"><DownloadIcon />Download Configuration </Button>
-                            {' '}
-                            <Button variant="link" onClick={() => router.push({pathname: "/auth/login", query: {next}})}>Go To Login</Button>
+                            <div className={"mt-4"} style={{textAlign: "center"}}>
+                                <Button className={"p-2 pl-3 pr-3 after-setup-btn"} onClick={() => window.open("/auth/login", "_blank")}>Go To Login</Button>
```

Comment:
This changes the current behaviour by creating a new tab, right?

Reply:
Yes, good catch! I was thinking it's better that to have the credentials still available in the previous tab.